pipelinewise/cli/alert_sender.py

```diff
@@ -116,13 +118,13 @@ def send_to_handler(
 
         # Initialise and create an alert handler object from the the alert handler spec
         handler = self.__init_handler_class(alert_handler)
-        handler.send(message=message, level=level, exc=exc)
+        handler.send(message=message, level=level, exc=exc, tap_slack_channel=tap_slack_channel)
```

Comment:
Are the arguments extendable? So if we decide to add same feature for victorops then it will look like this ?:
```
handler.send(message=message, level=level, exc=exc, tap_slack_channel=tap_slack_channel, tap_victorops_routing_key="stg")
```

Reply:
Let me change it in a better way.

- [ ] fix it